pkg/internal/export/otel/traces.go

```diff
@@ -321,6 +321,18 @@ func (r *TracesReporter) traceAttributes(span *request.Span) []attribute.KeyValu
 			semconv.NetPeerName(span.Host),
 			semconv.NetPeerPort(span.HostPort),
 		}
+	case request.EventTypeSQLClient:
+		operation := span.Method
+		if operation != "" {
+			attrs = []attribute.KeyValue{
+				semconv.DBOperation(operation),
+			}
+			table := span.Path
+			if table != "" {
+				attrs = append(attrs, semconv.DBSQLTable(table))
+				attrs = append(attrs, semconv.DBStatement(operation+" "+table))
```

Comment:
Since we don't actually have the full statement, we shouldn't set this attribute DBStatement. It adds extra data and it contains no new value we already haven't set. According to the OTel spec:

```
// DBStatementKey is the attribute Key conforming to the "db.statement"
// semantic conventions. It represents the database statement being
// executed.
```